q2as_json.cpp

```diff
@@ -160,6 +160,17 @@ static bool Q2AS_RegisterMutableJson(asIScriptEngine *engine)
     EnsureRegisteredMethodRaw("json_mutval", "bool get_is_false() const property", asMETHOD(q2as_yyjson_mut_val, is_false), asCALL_THISCALL);
     EnsureRegisteredMethodRaw("json_mutval", "bool get_is_bool() const property", asMETHOD(q2as_yyjson_mut_val,  is_bool), asCALL_THISCALL);
     EnsureRegisteredMethodRaw("json_mutval", "bool get_is_str() const property", asMETHOD(q2as_yyjson_mut_val,   is_str), asCALL_THISCALL);
+    
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_uint8() const property", asMETHOD(q2as_yyjson_mut_val, is_uint8), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_uint16() const property", asMETHOD(q2as_yyjson_mut_val, is_uint16), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_uint32() const property", asMETHOD(q2as_yyjson_mut_val, is_uint32), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_uint64() const property", asMETHOD(q2as_yyjson_mut_val, is_uint64), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_int8() const property", asMETHOD(q2as_yyjson_mut_val, is_int8), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_int16() const property", asMETHOD(q2as_yyjson_mut_val, is_int16), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_int32() const property", asMETHOD(q2as_yyjson_mut_val, is_int32), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_int64() const property", asMETHOD(q2as_yyjson_mut_val, is_int64), asCALL_THISCALL);
+    EnsureRegisteredMethodRaw("json_mutval", "bool get_is_double() const property", asMETHOD(q2as_yyjson_mut_val, is_double), asCALL_THISCALL);
```

Comment:
potentially missing is_float? not sure if we just want to check if there's no precision loss or something (like is_float() on 1.0 would be true, but is_float() on 0.33333678 would be false since it gets rounded).

Reply:
Fixed.